# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/fuse2/hotfix-int64.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

In file included from /usr/include/stdint.h:37,
                 from /usr/lib/gcc/aarch64-t2-linux-gnu/11.2.0/include/stdint.h:9,
                 from ../include/fuse_common.h:19,
                 from ../include/fuse_lowlevel.h:26,
                 from fuse_loop_mt.c:9:
/usr/include/bits/stdint-uintn.h:27:20: note: previous declaration of 'uint64_t' with type 'uint64_t' {aka 'long unsigned int'}
   27 | typedef __uint64_t uint64_t;
      |                    ^~~~~~~~

--- fuse-2.9.9/include/fuse_kernel.h.vanilla	2023-12-23 19:00:52.892000310 +0100
+++ fuse-2.9.9/include/fuse_kernel.h	2023-12-23 19:05:03.764000430 +0100
@@ -688,4 +689,10 @@
 	__u64	dummy4;
 };
 
+#undef __s64
+#undef __u64
+#undef __s32
+#undef __u32
+#undef __u16
+
 #endif /* _LINUX_FUSE_H */
